scripts/windows-recipe-smoke.ps1

```
@@ -0,0 +1,131 @@
+
+$workspace = Join-Path $env:RUNNER_TEMP "netsuke-windows-recipe-smoke-$PID"
+New-Item -ItemType Directory -Path $workspace | Out-Null
+Copy-Item -LiteralPath $Manifest -Destination (Join-Path $workspace 'Netsukefile')
+
+Push-Location $workspace
+try {
+    $env:NETSUKE_SMOKE_VALUE = 'value with spaces'
+
+    $discovery = & $Netsuke help targets 2>&1
+    if ($LASTEXITCODE -ne 0) {
+        throw "Target discovery failed with exit code $LASTEXITCODE: $discovery"
```

PSScriptAnalyzer (1.25.0)

[error] Variable reference is not valid. ':' was not followed by a valid variable name character. Consider using ${} to delimit the name.

(InvalidVariableReferenceWithDrive)